src/Behat/Behat/Definition/Call/DefinitionCall.php

```diff
@@ -20,6 +20,8 @@
  * Enhances environment call with definition information.
  *
  * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ *
```

Comment:
For this class, the inherited `getCallee(()` function will return a `Definition`, we can specify this here so that PHPStan is aware of this